libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -296,30 +297,23 @@ double Aircraft::rand_normal(double mean, double stddev)
 {
     static double n2 = 0.0;
     static int n2_cached = 0;
-    if (!n2_cached)
-    {
+    if (!n2_cached) {
         double x, y, r;
         do
         {
-            x = 2.0*rand()/RAND_MAX - 1;
-            y = 2.0*rand()/RAND_MAX - 1;
-
+            x = 2.0 * rand()/RAND_MAX - 1;
+            y = 2.0 * rand()/RAND_MAX - 1;
             r = x*x + y*y;
-        }
-        while (is_zero(r) || r > 1.0);
-        {
```

Comment:
http://web.archive.org/web/20150906061405/http://en.literateprograms.org/Box-Muller_transform_(C)

According to the source it is correct.